github/github.go

```diff
@@ -880,6 +880,23 @@ func cloneRequest(r *http.Request) *http.Request {
 	return r2
 }
 
+func formatDuration(d time.Duration) string {
```

Comment:
formatDuration could use a comment describing what it's meant to do.

```Go
// formatDuration formats d to look like "26m08s" or "8s".
```

Without a comment saying what the function does, one can only infer it from the implementation, and that makes it hard/impossible to ever improve the implementation, or figure out if a certain behavior is a bug or intended.